powerauth-admin/src/main/resources/application.properties

```diff
@@ -57,4 +57,6 @@ management.tracing.sampling.probability=1.0
 #management.endpoint.prometheus.enabled=true
 #management.prometheus.metrics.export.enabled=true
 
-spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.security.servlet.UserDetailsServiceAutoConfiguration
+spring.autoconfigure.exclude=\
+  org.springframework.boot.autoconfigure.security.servlet.UserDetailsServiceAutoConfiguration,\
```

Comment:
It suppresses WildFly a warning about `io.micrometer.core.instrument.binder.jersey.server.MetricsApplicationEventListener`.